{
  "comments": [
    {
      "unresolved": true,
      "key": {
        "uuid": "17348775_7a2e6b7e",
        "filename": "tests/src/test_report_manager.cpp",
        "patchSetId": 4
      },
      "lineNbr": 228,
      "author": {
        "id": 1000678
      },
      "writtenOn": "2022-01-26T08:18:42Z",
      "side": 1,
      "message": "CI is running address sanitizer which doesn\u0027t like exceptions and boost::asio. We investigated issue as false positive and as a workaround such tests are marked as DISABLED. We run tests locally with flag --gtest_also_run_disabled_tests.\n\nYou can find example in test below.",
      "range": {
        "startLine": 228,
        "startChar": 26,
        "endLine": 228,
        "endChar": 30
      },
      "revId": "a406f5197bf485d2cbf1175dc58dbb04a0827655",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4"
    }
  ]
}